.github/workflows/test-lint.yml

```diff
@@ -53,4 +53,4 @@ jobs:
         with:
           skip-pkg-cache: true
           skip-build-cache: true
-          args: --timeout=15m ${{ steps.submods.outputs.submods }}
+          args: --new-from-rev=HEAD~ --timeout=15m ${{ steps.submods.outputs.submods }}
```

Comment:
@katexochen is this what you had in mind regarding "only scan new code"? 
Not sure if I did it right, because a [manual trigger failed](https://github.com/edgelesssys/constellation/actions/runs/3471511080/jobs/5801096211). But it may be just because I used the wrong event to trigger the pipeline?

Reply:
```
with:
         ....

          # Optional: show only new issues if it's a pull request. The default value is `false`.
          # only-new-issues: true
```